packages/mui-base/src/TabsUnstyled/TabsContext.ts

```diff
@@ -1,31 +1,41 @@
 import * as React from 'react';
 
 export interface TabsContextValue {
-  /**
-   * Id used as a prefix for the current Tabs.
-   */
-  idPrefix: string | undefined;
   /**
    * The currently selected tab's value.
    */
-  value: number | string | false;
+  value: number | string | null;
   /**
    * Callback for setting new value.
    */
-  onSelected: (event: React.SyntheticEvent, value: number | string | false) => void;
+  onSelected: (event: React.SyntheticEvent | null, value: number | string | null) => void;
```

Comment:
I wonder if `onChange` wouldn't be better (more consistent with the other components)